### PR TITLE
Sourcebuffer

### PR DESCRIPTION
**Pull request by fwqaaq**
In localized document, If it is a document that has not been translated directly, it is recommended to use the latest English document directly and do not convert it.

### CODE REVIEW

File: files/zh-cn/web/api/sourcebuffer/appendbuffer/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  - Experimental
+  - MSE
+  - Media
+  - Media Source Extensions
+  - Method
+  - Reference
+  - SourceBuffer
+  - Video
+  - appendBuffer
+browser-compat: api.SourceBuffer.appendBuffer
```
Comment (yin1999) [inline files/zh-cn/web/api/sourcebuffer/appendbuffer/index.md:17]: please remove `BCD` metadata

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/api/sourcebuffer/mode/index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+{{domxref("SourceBuffer")}} 接口的 **`mode`** 属性用来控制媒体片段添加到 `SourceBuffer` 时的顺序是可以任意的还是有严格顺序的。
+
+两个可用的值：
+
+- `segments`: 媒体片段的时间戳决定段的播放顺序。这些片段可以按任意顺序添加到 sourcebuffer。
+- `sequence`: 媒体片段添加到 sourcebuffer 的顺序决定了它们的播放顺序。片段上的时间戳为遵守此顺序自动生成。
+
+mode 的值最初是在使用 mediasource.addsourcebuffer() 创建 sourcebuffer 时设置的。如果媒体片段上已经存在时间戳，则该值将设置为`segments`；如果没有，则该值将设置为`sequence`.
+
+如果 mode 初始值为 sequence 想要改为 segments 则会引发错误，必须以`sequence`模式维护现有段顺序。但是，可以将值从`segments 模式`改为`sequence 模式`。它意味着播放顺序将被固定，并会生成新的时间戳。
```
Comment (yin1999) [inline files/zh-cn/web/api/sourcebuffer/mode/index.md:29]:
```suggestion
如果 mode 初始值为 sequence 想要改为 segments 则会引发错误，必须以 `sequence` 模式维护现有段顺序。但是，可以将值从 `segments` 模式改为`sequence` 模式。它意味着播放顺序将被固定，并会生成新的时间戳。
```

**Review by yin1999 — COMMENTED** [review]
please fix and squash commit (except the first rename commit)

**Comment by Jack-Zhang-1314** [comment]
I used `squash`, but it becomes 6 commit now

**Comment by yin1999** [comment]
next time, please pull before making locale changes (for changes have been made in GitHub)

**Review by yin1999 — APPROVED** [review]

**Comment by Jack-Zhang-1314** [comment]
> next time, please pull before making locale changes (for changes have been made in GitHub)

* I tried it yesterday, I don't know if it's right

1. 需要回退的范围

```
git rebase -i 
# 将除了最新的提交的 pick 都改成 squash(s)
pick 4646e9c fw3
pick a0868c5 fw4
pick 911da7f fw5
pick 5c2e562 fw6

# Rebase 4073aa6..5c2e562 onto 4073aa6 (4 commands)
#
# Commands:

>>>>>>>>>>>>>>>>
pick 4646e9c fw3
s a0868c5 fw4
s 911da7f fw5
s 5c2e562 fw6
```

2. 显示所有的提交信息,有需要的可以更改

```bash
# This is the 1st commit message:

fw3

# This is the commit message #2:

fw4

# This is the commit message #3:

fw5

# This is the commit message #4:

fw6
```

3. 压缩这些提交信息

```bash
((d466032...)),这时候会进入到一个检出的提交中,这时候需要合并到主分支
git checkout -b dev
git checkout main
git rebase dev
git branch -D dev
```

> Or using `git rebase -i HEAD~n` directly, and the commit will not be checked out.

**Comment by yin1999** [comment]
https://backlog.com/git-tutorial/cn/stepup/stepup7_5.html

并不需要这里的第三步：

1. `git rebase -i HEAD~n`
2. 编辑rebase信息
3. 完成 rebase 后，直接 force push

**Comment by Jack-Zhang-1314** [comment]
> https://backlog.com/git-tutorial/cn/stepup/stepup7_5.html
> 
> 并不需要这里的第三步：
> 
> 1. `git rebase -i HEAD~n`
> 2. 编辑rebase信息
> 3. 完成 rebase 后，直接 force push

使用 `git rebase -i HEAD~n` 之后, 指针不会出现上面的游离态, 那我差不多明白了

**Pull request merged**